spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala

```diff
@@ -1352,15 +1352,11 @@ object CometSparkSessionExtensions extends Logging {
     org.apache.spark.SPARK_VERSION >= "4.0"
   }
 
-  def isComplexTypeReaderEnabled(conf: SQLConf): Boolean = {
+  def usingParquetExec(conf: SQLConf): Boolean = {
```

Comment:
`usingParquetExec` isn't it too generic? `usingNativeScan` or `usingNativeDataFusionScan`?

Reply:
Thanks. I renamed to `usingDataFusionParquetExec`.